### PR TITLE
feat: Gamify setup

### PR DESCRIPTION
**Pull request by sanderelderman**
Setup the gamify project by generating an empty Angular and NestJS app

### CODE REVIEW

File: apps/gamify/api/project.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{
```
Comment (mschilling) [inline apps/gamify/api/project.json:1]: Info: including the 'project.json' file in the project itself, is the new 'nx best way to do it'. So great it's here :-)

For some generators, when creating an app or lib, you have to tick on this option. So this is just a heads-ups to be aware of it.

File: apps/gamify/web-app/proxy.conf.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "/api": {
+    "target": "http://localhost:3333",
```
Comment (mschilling) [inline apps/gamify/web-app/proxy.conf.json:3]: this might change

File: package.json [hunk 3]
```diff
@@ -53,14 +72,17 @@
...
+    "eslint-plugin-cypress": "^2.10.3",
     "jest": "27.5.1",
+    "jest-preset-angular": "11.1.1",
+    "nx": "13.9.5",
     "prettier": "2.5.1",
     "prisma": "^3.10.0",
     "ts-jest": "27.1.3",
-    "typescript": "4.6.2",
-    "nx": "13.9.5"
+    "typescript": "^4.5.2"
```
Comment (mschilling) [inline package.json:85]: One of the things I dislike, when using nx<>angular -> downgrade or pinning the typescript version.
(But this is the way how to do this 👍🏽 )

**Review by mschilling — APPROVED** [review]
Nice work! 🚀

File: package.json (hunk 3 above)
Comment (cassshh) [reply package.json:85]: :yea: 😠

**Review by cassshh — APPROVED** [review]

**Pull request merged**